packages/next/src/server/app-render/app-render.tsx

```diff
@@ -890,6 +922,7 @@ export async function renderToHTMLOrFlight(
       if (!Component) {
         return {
           Component: () => <>{parallelRouteComponents.children}</>,
+          assets: styles,
```

Comment:
rename `assets` to `styles`? As in the above functions `assets` including both `styles` and `preloads`. `preloads` are not returned along with it here